server/app.js

```diff
@@ -4,7 +4,7 @@ import logger from 'morgan';
 import bodyParser from 'body-parser';
 import cors from 'cors';
 
-import router from './routes/routes';
+import routes from './routes/index';
```

Comment:
It's okay to import your routes like so: `import routes from './routes'` since you are using a default export in your index.

Reply:
Fixed